INSTALL

```diff
@@ -0,0 +1,213 @@
+Prerequisites:
+==============
+1. Python Git: sudo apt-get install python-git
+2. Google repo: https://source.android.com/source/downloading
+3. Checked with Python v 2.7.12, but other should also work
+
+About:
+======
+Gen3-test product is based on Renesas's BSP and Xen hypervisor. It contains a set of features required to have
+both Display and GPU support in several domains at the same time (device passthrough, PVDRM, VGPU, etc).
+
+There are three domains which are running on top of Xen:
+1. Generic machine independent control domain named "domain 0". This "initramfs" based domain is responsible
```

Comment:
not sure why you need quotes around initramfs

Reply:
ok